CONTRIBUTING.md

```diff
@@ -2,14 +2,33 @@
 
 #### 1. Create an issue to make sure its something that should be done.
 
+Before submitting a Github issue, please make sure to
+
+- Search for [existing Github issues](https://github.com/RevenueCat/purchases-ios/issues)
+- Review our [Help Center](https://support.revenuecat.com/hc/en-us)
+- Read our [docs.revenuecat.com](https://docs.revenuecat.com/)
+
+## Common project specific issues
+
+There are certain project specific issues that are commonly misinterpreted as bugs.
+
+- [Offerings, products, or available packages are empty](https://support.revenuecat.com/hc/en-us/articles/360041793174)
+- [Invalid Play Store credentials errors](https://support.revenuecat.com/hc/en-us/articles/360046398913)
+- [Unable to connect to the App Store (STORE_PROBLEM) errors](https://support.revenuecat.com/hc/en-us/articles/360046399333)
+
+For support I'd recommend our [online community](https://spectrum.chat/revenuecat), [StackOverflow](https://stackoverflow.com/tags/revenuecat/) and/or [Help Center](https://support.revenuecat.com/hc/en-us) 👍
+
+If you have a clearly defined bug (with a [Minimal, Complete, and Reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)) that is not specific to your project, follow the steps in the GitHub Issue template to file it with RevenueCat without removing any of the steps. For SDK-related bugs, make sure they can be reproduced on a physical device, not a simulator (there are simulator-specific problems that prevent purchases from working).
+
 #### 2. Create a fork/branch.
 
 #### 3. Do your work.
+
 Be sure to adhere to the prevailing style of the project.
 
 #### 4. Write tests for your fix/new functionality.
 
-You can run the tests by hitting Cmd+U. The tests are written swift 
+You can run the tests by hitting Cmd+U. The tests are written swift
```

Comment:
Lol this copy is a bit broken, I never saw it before. 
Could you update this to: 
```
You can run the tests by selecting the All Tests Scheme in Xcode and hitting `Cmd+U`. 
The tests are written in Swift, using XCTest and [Nimble](https://github.com/quick/nimble). 
```

Reply:
just added a commit to update!